config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users, path: 'auth', path_names: { sign_in: 'login', sign_out: 'logout', 
```

Comment:
We're not going to need the paths for the controllers, we don't want extra paths we won't use